lib/routes/github/private-feed.ts

```diff
@@ -0,0 +1,230 @@
+                },
+                {
+                    label: 'Create events',
+                    value: 'create',
+                },
+                {
+                    label: 'Delete events',
+                    value: 'delete',
+                },
+                {
+                    label: 'Fork events',
+                    value: 'fork',
+                },
+                {
+                    label: 'Issue comment events',
+                    value: 'issuecomm',
+                },
+                {
+                    label: 'Member events',
+                    value: 'member',
+                },
+                {
+                    label: 'Pull request events',
+                    value: 'pr',
+                },
+                {
+                    label: 'Pull request review comment events',
+                    value: 'prcomm',
+                },
+                {
+                    label: 'Public events',
+                    value: 'public',
+                },
+                {
+                    label: 'Push events',
+                    value: 'push',
+                },
+                {
+                    label: 'Release events',
+                    value: 'release',
+                },
+                {
+                    label: 'Watch events (stars)',
+                    value: 'star',
+                },
+            ],
+        },
+    },
+    features: {
+        requireConfig: [
+            {
+                name: 'GITHUB_ACCESS_TOKEN',
+                optional: true,
+                description: 'GitHub access token to access private events',
+            },
+        ],
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['github.com/:user'],
+            target: '/privatefeed/:user',
+        },
+    ],
+    name: 'User\'s Feed',
+    maintainers: ['RtYkk'],
+    handler,
+};
+
+function formatEventItem(event: any) {
+    const { type, actor, repo, payload, created_at } = event;
+
+    let title = '';
+    let description = '';
+    let link = '';
+
+    switch (type) {
+        case 'PushEvent':
+            title = `${actor.login} pushed to ${repo.name}`;
+            description = `Pushed ${payload.commits?.length || 0} commit(s) to ${repo.name}`;
+            link = `https://github.com/${repo.name}`;
+            if (payload.commits && payload.commits.length > 0) {
+                description += `<br><strong>Latest commit:</strong> ${payload.commits.at(-1).message}`;
+            }
+            break;
+        case 'PullRequestEvent':
+            title = `${actor.login} ${payload.action} a pull request in ${repo.name}`;
+            description = `PR: ${payload.pull_request?.title || 'Unknown'}`;
+            link = payload.pull_request?.html_url || `https://github.com/${repo.name}`;
```

Comment:
Since RSSHub will use `link` as `guid` if it's not defined. Please do specify a `guid` so that RSS reader can distinguish someone open a PR and closed a PR as they both share the same link or similar as all cases use the repo link only.